lib/__tests__/standalone-fix.test.mjs

```diff
@@ -117,52 +94,6 @@ it("doesn't fix with multiple scoped stylelint-disable commands", async () => {
 	expect(result.output).toBe(code);
 });
 
-it("the color-hex-length rule doesn't fix with scoped stylelint-disable commands", async () => {
-	const result = await standalone({
-		code: stripIndent`
-			/* stylelint-disable color-hex-length */
-			a {
-				color: #ffffff;
-			}`,
-		config: {
-			rules: {
-				indentation: 2,
-				'color-hex-length': 'short',
-			},
-		},
-		fix: true,
-	});
-
-	expect(result.output).toBe(stripIndent`
-		/* stylelint-disable color-hex-length */
-		a {
-		  color: #ffffff;
-		}`);
-});
-
-it("the indentation rule doesn't fix with scoped stylelint-disable commands", async () => {
-	const result = await standalone({
-		code: stripIndent`
-			/* stylelint-disable indentation */
-			a {
-				color: #ffffff;
-			}`,
-		config: {
-			rules: {
-				indentation: 2,
-				'color-hex-length': 'short',
-			},
-		},
-		fix: true,
-	});
-
-	expect(result.output).toBe(stripIndent`
-		/* stylelint-disable indentation */
-		a {
-			color: #fff;
-		}`);
-});
-
```

Comment:
These feel like duplicate tests, but if there's a general principle we're trying to test, I can rewrite these too!

Reply:
I don't think we shouldn't remove these cases. Could you rewrite them by using existing rules? And, we should make `describe` more generic.

Reply:
Sounds good; I've tried to replicate what feels like the spirit of the test in https://github.com/stylelint/stylelint/pull/6979/commits/3556d6d7725e08bd072fdb58fdb74d4725001cc2; let me know if this is what we were looking for!